requirements.txt

```diff
@@ -1,6 +1,6 @@
 qiskit>=0.32
 mthree>=0.17.2
-git+https://github.com/PennyLaneAI/pennylane.git
+pennylane>=0.20
```

Comment:
@antalszava, do you think the failure is because we actually require the latest version of PL for this ?

Reply:
Yes, see here https://github.com/PennyLaneAI/pennylane-qiskit/pull/177/files.

Changed the test to consider this.

Reply:
Ahh I see !

Reply:
Nice catch!